crates/turborepo-lib/src/commands/mod.rs

```diff
@@ -84,6 +85,17 @@ impl CommandBase {
         let global_config_path = config_dir.join("turborepo").join("config.json");
         AbsoluteSystemPathBuf::try_from(global_config_path).map_err(ConfigError::PathError)
     }
+    /// Returns the path to the global auth file (auth.json).
+    fn global_auth_path(&self) -> Result<AbsoluteSystemPathBuf, ConfigError> {
+        #[cfg(test)]
+        if let Some(global_auth_path) = self.global_auth_path.clone() {
+            return Ok(global_auth_path);
```

Comment:
Very much a nit since `None.clone()` doesn't actually allocate any memory, but as a rule of thumb if you can avoid a `clone` call (or any function call which might be costly). Small change that borrows the `Option` and we only `clone` if there's something present.
```suggestion
        if let Some(global_auth_path) = &self.global_auth_path {
            return Ok(global_auth_path.clone());
```